src/Microsoft.ML.ImageAnalytics/ImageGrayscaleTransform.cs

```diff
@@ -0,0 +1,171 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Drawing;
+using System.Drawing.Imaging;
+using System.Text;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.CommandLine;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Runtime.ImageAnalytics;
+
+[assembly: LoadableClass(ImageGreyscaleTransform.Summary, typeof(ImageGreyscaleTransform), typeof(ImageGreyscaleTransform.Arguments), typeof(SignatureDataTransform),
+    ImageGreyscaleTransform.UserName, "ImageGreyscaleTransform", "ImageGreyscale")]
+
+[assembly: LoadableClass(ImageGreyscaleTransform.Summary, typeof(ImageGreyscaleTransform), null, typeof(SignatureLoadDataTransform),
+    ImageGreyscaleTransform.UserName, ImageGreyscaleTransform.LoaderSignature)]
+
+namespace Microsoft.ML.Runtime.Data
+{
+    // REVIEW: Rewrite as LambdaTransform to simplify.
+    public sealed class ImageGreyscaleTransform : OneToOneTransformBase
+    {
+
+        public sealed class Column : OneToOneColumn
+        {
+            public static Column Parse(string str)
+            {
+                var res = new Column();
+                if (res.TryParse(str))
+                    return res;
+                return null;
+            }
+
+            public bool TryUnparse(StringBuilder sb)
+            {
+                Contracts.AssertValue(sb);
+                return TryUnparseCore(sb);
+            }
+
+        }
+
+        public class Arguments : TransformInputBase
+        {
+            [Argument(ArgumentType.Multiple | ArgumentType.Required, HelpText = "New column definition(s) (optional form: name:src)", ShortName = "col", SortOrder = 1)]
+            public Column[] Column;
+        }
+
+        internal const string Summary = "Convert image into grayscale.";
+
+        internal const string UserName = "Image Greyscale Transform";
+        public const string LoaderSignature = "ImageGreyscaleTransform";
+        private static VersionInfo GetVersionInfo()
+        {
+            return new VersionInfo(
+                modelSignature: "IMGGREY ",
+                verWrittenCur: 0x00010001, // Initial
+                verReadableCur: 0x00010001,
+                verWeCanReadBack: 0x00010001,
+                loaderSignature: LoaderSignature);
+        }
+
+        private const string RegistrationName = "ImageGreyscale";
+
+        /// <summary>
+        /// Public constructor corresponding to SignatureDataTransform.
+        /// </summary>
```

Comment:
I know we use this allover, and it's plenty meaningful as a code comment, but it will appear in the docs site as well, where might confuse users. Maybe turn it into a code comment removing the triple ///, and <summary>. 
Ideally add a meaningful line for the actual documentation.